diag_manager/fms_diag_output_buffer.F90

```diff
@@ -213,33 +221,34 @@ end subroutine set_buffer_id
 
 !> Remaps 0-5d data buffer from the given object onto a 5d array pointer.
 !> @returns a 5D remapped buffer, with 1:1 for any added dimensions.
-function remap_buffer(buffobj, field_name, has_diurnal_axis)
+function remap_buffer(buffobj, field_name, file_name, has_diurnal_axis)
   class(fmsDiagOutputBuffer_class), target, intent(inout) :: buffobj !< any dimension buffer object
   class(*), pointer                                 :: remap_buffer(:,:,:,:,:)
   character(len=*), intent(in)                      :: field_name !< name of field for error output
+  character(len=*), intent(in)                      :: file_name !< name of field for error output
   logical, intent(in) :: has_diurnal_axis !< true if the buffer has diurnal axis
 
   ! get num dimensions from type extension
   select type (buffobj)
     type is (outputBuffer0d_type)
       if (.not. allocated(buffobj%buffer)) call mpp_error(FATAL, "remap_buffer: buffer data not yet allocated" // &
-                                                                 "for field:" // field_name)
+                                                                 "for field:" // field_name // "in file:" // file_name)
```

Comment:
I think `field_name` and `file_name` need to be `trim(field_name)` and `trim(file_name)`